packages/workers-shared/router-worker/src/analytics.ts

```diff
@@ -0,0 +1,70 @@
+// This will allow us to make breaking changes to the analytic schema
+const VERSION = 1;
+
+export enum DISPATCH_TYPE {
+	ASSETS = "asset",
+	WORKER = "worker",
+}
+
+// When adding new columns please update the schema
+type Data = {
```

Comment:
Good schema! When we add more complex routing config options, I'll be tempted to add some sort of "routing evaluation time" thing, but we can do that later of course!

Reply:
Credit goes to Walshy-- this was largely stolen from analytics in pages :)